src/OrbitGgp/include/OrbitGgp/Client.h

```diff
@@ -17,21 +18,34 @@
 #include "OrbitBase/Result.h"
 #include "SshInfo.h"
 
+namespace {
+
+constexpr const char* kDefaultGgpProgram{"ggp"};
+constexpr std::chrono::milliseconds kDefaultTimeout{10'000};
+
+}  // namespace
```

Comment:
Clang-tidy is upset about anonymous namespaces in header files and that variable definitions can lead to ODR violations. I would appreciate a suggestion on how to make this better.

Reply:
If you want to declare and define variables in a header file you want to mark them `inline` which avoids the ODR issue.
Since `constexpr` implies `inline` you can just leave out the anonymous namespace.

Reply:
We talked about the constexpr and namespace situation before. I did forget about it :/ Thank you for the explanation!